app/views/talks/_card_horizontal.html.erb

```diff
@@ -1,21 +1,25 @@
-<%# locals: (talk:, current_talk: nil, compact: false) -%>
+<%# locals: (talk:, current_talk: nil, compact: false, turbo_frame: nil) -%>
 
 <% active = talk == current_talk %>
-<%= link_to talk_path(talk), class: class_names("w-full flex items-center gap-4 p-2 rounded-lg hover:bg-gray-200", "bg-gray-200": active),
+<%= link_to talk_path(talk),
+      class: class_names("w-full flex items-center gap-4 p-2 rounded-lg hover:bg-gray-200 group", active: active, watched: talk.watched?),
       id: dom_id(talk, :card_horizontal),
-      data: {talk_horizontal_card: true, scroll_into_view_target: (active ? "active" : nil), controller: "tooltip", tooltip_content_value: talk.title} do %>
+      data: {talk_horizontal_card: true,
+             scroll_into_view_target: (active ? "active" : nil),
+             talks_navigation_target: "card",
+             action: "talks-navigation#setActive",
+             turbo_frame: turbo_frame,
+             controller: "tooltip", tooltip_content_value: talk.title} do %>
   <div class="flex aspect-video shrink-0 relative w-20">
     <%= image_tag talk.thumbnail_sm, srcset: ["#{talk.thumbnail_sm} 2x"], id: dom_id(talk), class: "w-full h-auto object-cover border rounded", loading: :lazy %>
 
-    <% if active %>
-      <div class="absolute inset-0 bg-black/50 flex justify-center items-center rounded">
-        <%= fa("play", size: :sm, class: "fill-white") %>
-      </div>
-    <% elsif talk.watched? %>
-      <button data-tip="Talk Watched" class="tooltip tooltip-right absolute inset-0 bg-black/50 flex justify-center items-center rounded">
-        <%= fa("circle-check", size: :sm, class: "fill-white") %>
-      </button>
-    <% end %>
+    <div class="absolute inset-0 bg-black/50 justify-center items-center rounded hidden group-[.active]:flex">
+      <%= fa("play", size: :sm, class: "fill-white") %>
+    </div>
+
+    <button data-tip="Talk Watched" class="tooltip tooltip-right absolute inset-0 bg-black/50 justify-center items-center rounded hidden group-[.watched]:flex group-[.active]:hidden">
+      <%= fa("circle-check", size: :sm, class: "fill-white") %>
+    </button>
```

Comment:
Does this account for the case where you mark a talk as watched so that it will add the checkmark to the cards in the sidebar?

Reply:
currently it does not update the side bar. I will address that in a follow up PR